src/main/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/LoansToDirectorsAdditionalInfoServiceImpl.java

```diff
@@ -1,70 +1,78 @@
 import uk.gov.companieshouse.api.model.ApiResponse;
 import uk.gov.companieshouse.api.model.accounts.smallfull.loanstodirectors.AdditionalInformationApi;
 import uk.gov.companieshouse.web.accounts.api.ApiClientService;
 import uk.gov.companieshouse.web.accounts.exception.ServiceException;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.loanstodirectors.LoansToDirectorsAdditionalInfo;
 import uk.gov.companieshouse.web.accounts.service.smallfull.LoansToDirectorsAdditionalInfoService;
-import uk.gov.companieshouse.web.accounts.service.smallfull.SmallFullService;
+import uk.gov.companieshouse.web.accounts.transformer.smallfull.loanstodirectors.LoanToDirectorsAdditionalInfoTransformer;
 import uk.gov.companieshouse.web.accounts.util.ValidationContext;
 import uk.gov.companieshouse.web.accounts.validation.ValidationError;
 import uk.gov.companieshouse.web.accounts.validation.helper.ServiceExceptionHandler;
 
-import java.util.ArrayList;
-import java.util.List;
-
 @Service
 public class LoansToDirectorsAdditionalInfoServiceImpl implements LoansToDirectorsAdditionalInfoService {
 
     @Autowired
     private ApiClientService apiClientService;
 
-    @Autowired
-    private SmallFullService smallFullService;
-
     @Autowired
     private ServiceExceptionHandler serviceExceptionHandler;
 
     @Autowired
     private ValidationContext validationContext;
 
+    @Autowired
+    private LoanToDirectorsAdditionalInfoTransformer loanToDirectorsAdditionalInfoTransformer;
+
     private static final UriTemplate ADDITIONAL_INFORMATION_URI =
             new UriTemplate("/transactions/{transactionId}/company-accounts/{companyAccountsId}/small-full/notes/loans-to-directors/additional-information");
 
     private static final String RESOURCE_NAME = "loans to directors additional information";
 
     @Override
-    public AdditionalInformationApi getAdditionalInformation(String transactionId, String companyAccountsId) throws ServiceException {
+    public LoansToDirectorsAdditionalInfo getAdditionalInformation(String transactionId, String companyAccountsId) throws ServiceException {
 
         ApiClient apiClient = apiClientService.getApiClient();
 
         String uri = ADDITIONAL_INFORMATION_URI.expand(transactionId, companyAccountsId).toString();
 
         try {
-            return apiClient.smallFull().loansToDirectors().additionalInformation().get(uri).execute().getData();
+            AdditionalInformationApi additionalInformationApi = apiClient.smallFull().loansToDirectors().additionalInformation().get(uri).execute().getData();
+            
+            return loanToDirectorsAdditionalInfoTransformer.mapLoansToDirectorsAdditionalInfoToWeb(additionalInformationApi);
         } catch (URIValidationException e) {
             serviceExceptionHandler.handleURIValidationException(e, RESOURCE_NAME);
         } catch (ApiErrorResponseException e) {
+            if(e.getStatusCode() == HttpStatusCodes.STATUS_CODE_NOT_FOUND) {
```

Comment:
Remove this block and instead of returning `null` at line 63, `return new LoansToDirectorsAdditionalInfo()`